site/sfguides/src/servicenow_to_snowflake_connector/servicenow_to_snowflake_connector.md

```diff
@@ -0,0 +1,364 @@
+### Prerequisites
+- ServiceNow account with administrator's rights.
+- ORGADMIN rights to Accept the Terms of Service in the Snowflake Marketplace.
+- ACCOUNTADMIN rights on the Snowflake account where you will install the connector.
+
+### What You’ll Learn 
+- How to set up the Snowflake Connector for ServiceNow.
+- How to ingest ServiceNow data into Snowflake
+- How to stop the connector to avoid unnecessary costs in a development environment.
+### What You’ll Need 
+- A [Snowflake](https://snowflake.com/) Account 
+- A [ServiceNow](https://developer.servicenow.com/dev.do/) developer account
+### What You’ll Build 
+A ServiceNow to Snowflake ingestion data flow.
+
+<!-- ------------------------ -->
+## Set up the ServiceNow Developer Instance
+Duration: 5
+
+If you do not want to test this connector on your ServiceNow account, no problem, this step explains how to set up a developer instance! 
+
+1. Go to the [ServiceNow developer website](https://developer.servicenow.com), and create a developer user.
+
+1. Log on to the developer website with your newly created user and select **Create an Instance**. 
+1. Choose an instance type. You receive an email with your instance URL, and admin user and password. 
+
+Deployment is usually pretty quick, around five minutes. But, while you wait let's go to the next step and configure Snowflake!
+## Create and set up the Snowflake Account
+
+### Create the Snowflake Account
+If you do not have a Snowflake account, no problem, you can get a free trial  at [snowflake.com](https://www.snowflake.com/en/).Select **Start for Free** and follow the instructions. You will probably want to deploy in the same region as the ServiceNow deployment, but this is not a requirement for the lab. 
+### Accept the Terms & Conditions
+Duration: 1
+1. Log on to your Snowflake account through the Snowsight web interface and change to the **orgadmin** role. 
+1. Select “Admin -> Billing & Terms”.
+4. In the “Snowflake Marketplace” section, review the Consumer Terms of Service.
+5. If you agree to the terms, select “Accept Terms & Conditions”.
+
+### Set up a Virtual Warehouse
+Duration: 3
+
+You'll need some compute for the connector, so let's set up a virtual warehouse to do that. A second virtual warehouse will be created automatically in the configure section.
+
+Change to the **accountadmin** role.
+1. Navigate to Admin -> Warehouses and select **+ Warehouse**. 
+2. Name the  vitural warehouse **SERVICENOW_CONNECTOR_WH**, size XS, and, leaving the defaults, select **Create Warehouse**. 
+
+### Install the ServiceNow connector
+Duration: 3
+
+The connector, the first of its kind to be deployed on Snowflake's native apps framework, is delivered through the Snowflake marketplace, available to all Snowflake customers instantly. Once chosen, it is installed into your account as a database with several schemas, tables, views, and protected stored procedures. 
```

Comment:
```suggestion
The connector, the first of its kind to be deployed on Snowflake's native apps framework, is delivered through the Snowflake Marketplace, available to all Snowflake customers instantly. Once chosen, it is installed into your account as a database with several views and stored procedures. 
```
Regarding the database contents - it is true that there are several schemas and tables but only one schema and no tables are visible to the customer. Also, what did you mean by `protected` stored procedure?